integrations/adk-py/examples/hello_eval.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+from typing import Any
+
+from braintrust.framework import EvalAsync
+from braintrust_adk import setup_braintrust
+from hello import main as hello
+
+
+async def main():
+    async def task(input: Any):
+        setup_braintrust()
+        return await hello(input)
+
+    await EvalAsync(
```

Comment:
verifies that evals + integration continues to work as expected:

<img width="819" height="803" alt="Image" src="https://github.com/user-attachments/assets/024fb8da-526f-434e-86fd-1d2bdfb76e7d" />